README.md

```diff
@@ -26,81 +27,48 @@ These names are used interchangably in this repository.
 - The architecture is subject to change in the future to lessen the dependence
   on MySQL.
 
+
 ## Test Harness
 
 A test harness is provided in `./harness.sh`. The test harness uses tmux to start
 a dcrd node, multiple dcrwallet and stakepoold instances, and finally a dcrstakepool
 instance. It uses hard-coded wallet seeds and pubkeys, and as a result it is only
 suitable for use on testnet. Further documentation can be found in `./harness.sh`.
 
+
 ## Requirements
 
-- [Go](https://golang.org) 1.11.13 or newer (1.12 is recommended).
+- [Go](https://golang.org) 1.11.13 or newer (1.13 is recommended).
 - MySQL
 - Nginx or other web server to proxy to dcrstakepool
 
-## Installation
-
-### Build from Source
-
-Building or updating from source requires only an installation of Go
-([instructions](https://golang.org/doc/install)). It is recommended to add
-`$GOPATH/bin` to your `PATH` at this point.
-
-Clone the dcrstakepool repository into any folder and follow the instructions
-below for your version of Go.
-
-#### Building with Go 1.12
-
-If building in a folder
-under `GOPATH`, it is necessary to explicitly build with modules enabled:
-
-```bash
-    GO111MODULE=on go build
-```
 
-If building outside of `GOPATH`, modules are automatically enabled, and `go
-build` is sufficient.
+## Installation
 
-The `go` tool will process the source code and automatically download
-dependencies. If the dependencies are configured correctly, there will be no
-modifications to the `go.mod` and `go.sum` files.
+### Build from source (all platforms)
 
-### Components
+Building or updating from source requires the following build dependencies:
 
-The frontend server (dcrstakepool) and the backend daemon (stakepoold) are built
-separately. Since module-enabled builds no longer require building under
-`$GOPATH`, the following instructions use the placeholder
-`{{YOUR_GO_MODULE_PATH}}` to refer to wherever you checkout your Go code.
+- **Go 1.11 ~ 1.13**
 
-#### Frontend - dcrstakepool
+  Installation instructions can be found here: https://golang.org/doc/install.
+  It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
 
-Build dcrstakepool and copy it to the web server.
+- **Git**
 
-```bash
-$ cd {{YOUR_GO_MODULE_PATH}}/github.com/decred/dcrstakepool
-$ go build
-```
+  Installation instructions can be found at https://git-scm.com
 
-#### Backend - stakepoold
+To build and install from a checked-out repo, run `go install . ./backend/stakepoold/...`
```

Comment:
We dont need `...` here as there is no further dirs to work through. `go install . ./backend/stakepoold` is enough

Reply:
I was thinking the dots pulled dependencies, but maybe I'm thinking of go get, and maybe that's not even an issue with modules?

Reply:
`...` is essentially for recursion down into the dir tree.

The command `go help packages` provides a nice explanation